eventing/default-channels.md

```diff
@@ -0,0 +1,129 @@
+# Default Channels
+
+The default channel configuration allows channels to be created without
+specifying a provisioner. This leaves the selection of channel provisioner and
+properties up to the operator. The operator controls the default settings via a
+ConfigMap.
+
+## Creating a default channel
+
+To create a default channel, use the same procedure as for creating a normal
```

Comment:
This suggests that the normal mode of operation is to specify a `spec.provisioner`. Should we turn this around, and say that the normal mode is to not specify a provisioner, and to rely on this default?

Reply:
I just removed the clause about "use the same procedure...". Seems clearer. I agree that the normal mode should be to rely on the default.